methods/ccf.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "ccf",
+    "status": "registered",
+    "verifiableDataRegistry": "Ledger agnostic",

```

Comment:
I think this is possibly not correct, the ledger is a CCF here right?

Or is the idea for this method to also support Trillion and QLDB?

Reply:
You are correct this should have been CCF.